client/my-sites/guided-transfer/host-select.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import Button from 'components/button';
+
+class HostSelect extends Component {
+	showHost( hostSlug ) {
+		// Temporary page redirect with a # in it.
+		page( `/settings/export/${this.props.siteSlug}/guided#${hostSlug}` );
```

Comment:
We could probably change this to `/settings/export/guided/${hostSlug}/${this.props.siteSlug}` as per [this comment](https://github.com/Automattic/wp-calypso/pull/6200#discussion_r69292392)